message/obs/likes.js

```diff
@@ -14,20 +14,21 @@ exports.create = function (api) {
   var likesLookup = {}
   return nest({
     'sbot.hook.feed': (msg) => {
-      if (msg.value && typeof msg.value.content === 'string') {
+      if (!(msg && msg.value && msg.value.content)) return
+      if (typeof msg.value.content === 'string') {
         msg = api.message.sync.unbox(msg)
+        if (!msg) return
```

Comment:
This was a massive gotcha!  turns out unbox returns `undefined` if it can't unbox